packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -2,62 +2,99 @@ import Component from '@glimmer/component';
 import { fn } from '@ember/helper';
 import { on } from '@ember/modifier';
 import { action } from '@ember/object';
-import { RenderedLinksToCard } from './stack-item';
+import { RenderedCardForOverlayActions } from './stack-item';
 import { velcro } from 'ember-velcro';
 import { Actions } from '@cardstack/runtime-common';
 import { IconButton } from '@cardstack/boxel-ui';
 import cn from '@cardstack/boxel-ui/helpers/cn';
 import { type TrackedArray } from 'tracked-built-ins';
 import type { MiddlewareState } from '@floating-ui/dom';
 import type { Card } from 'https://cardstack.com/base/card-api';
+import { cardTypeDisplayName } from '@cardstack/runtime-common';
+import { and, eq, not } from '@cardstack/host/helpers/truth-helpers';
+import { bool } from '@cardstack/boxel-ui/helpers/truth-helpers';
 
 interface Signature {
   Args: {
-    renderedLinksToCards: RenderedLinksToCard[];
+    renderedCardsForOverlayActions: RenderedCardForOverlayActions[];
     publicAPI: Actions;
     toggleSelect?: (card: Card) => void;
     selectedCards?: TrackedArray<Card>;
   };
 }
 
 export default class OperatorModeOverlays extends Component<Signature> {
+  isEmbeddedCard(renderedCard: RenderedCardForOverlayActions) {
+    return (
+      renderedCard.fieldType === 'contains' ||
+      renderedCard.fieldType === 'linksTo'
+    );
+  }
+
```

Comment:
`fieldType` possibly undefined. Shall we explicitly express that in this function.

Reply:
If it's `undefined` then it's not really embedded right? In that case I think we're fine